pkg/jobrunaggregator/jobrunaggregatorlib/ci_data_client.go

```diff
@@ -141,6 +144,84 @@ LIMIT 1
 	return lastJobRun, nil
 }
 
+func (c *ciDataClient) GetBackendDisruptionStatisticsByJob(ctx context.Context, jobName string) ([]jobrunaggregatorapi.BackendDisruptionStatisticsRow, error) {
+	rows := make([]jobrunaggregatorapi.BackendDisruptionStatisticsRow, 0)
+
+	queryString := c.dataCoordinates.SubstituteDataSetLocation(`
+SELECT
+    p95.BackendName,
+    p95.P95,
+    mean.Mean 
+FROM
+    (
+        SELECT
+            BackendName,
+            ANY_VALUE(P95) AS P95
+            FROM (
+                SELECT
+                    BackendName,
+                    PERCENTILE_CONT(BackendDisruption.DisruptionSeconds, 0.95) OVER(PARTITION BY BackendDisruption.BackendName) AS P95,
+                FROM
+                    DATA_SET_LOCATION.BackendDisruption as BackendDisruption
+                INNER JOIN
+                    DATA_SET_LOCATION.BackendDisruption_JobRuns as JobRuns on JobRuns.Name = BackendDisruption.JobRunName
+                WHERE
+                    JobRuns.StartTime BETWEEN TIMESTAMP_SUB(CURRENT_TIMESTAMP(), INTERVAL 10 DAY)
+                AND
+                    TIMESTAMP_SUB(CURRENT_TIMESTAMP(), INTERVAL 3 DAY)
+                AND
+                    JobRuns.JobName = @JobName
+            )
+            GROUP BY
+                BackendName
+      ) p95
+LEFT JOIN
+    (
+        SELECT
+            BackendName,
+            AVG(BackendDisruption.DisruptionSeconds) as Mean
```

Comment:
Can't do this in the same select as the percentile?

Reply:
I couldn't get it to work